components/typography/demo/basic.md

```diff
@@ -31,7 +31,7 @@ ReactDOM.render(
     </Paragraph>
     <Paragraph>
       After massive project practice and summaries, Ant Design, a design language for background
-      applications, is refined by Ant UED Team, which aims to
+      applications, is refined by Ant UED Team, which aims to&nbsp;
```

Comment:
```suggestion
      applications, is refined by Ant UED Team, which aims to <Text strong>
```

Reply:
Thank you so much. First time I thought it may destroy the code structure, so add '&nbsp', so sorry.